packages/eslint-plugin/rules/data-no-store-string-literals.js

```diff
@@ -198,6 +198,7 @@ function getFixes( fixer, context, callNode ) {
 module.exports = {
 	meta: {
 		type: 'problem',
+		hasSuggestions: true,
```

Comment:
This is now necessary (since ESLint v8) if a rule provides suggestions: https://eslint.org/docs/user-guide/migrating-to-8.0.0